src/bin/copy_builder_test_machine.js

```diff
@@ -0,0 +1,21 @@
+import { testerConfig, testerOptions } from '../machines/builder_route/tester.config';
+// eslint-disable-next-line node/no-unpublished-import
+import clip from 'clipboardy';
+
+function stringify (obj) {
```

Comment:
I'd rename this to `stringifyObj`, moving it to `src/node_modules/svizzle/utils/obj-string.js` for later evaluation